pkgs/misc/cups/default.nix

```diff
@@ -96,7 +101,8 @@ stdenv.mkDerivation rec {
       for f in "$out"/lib/systemd/system/*; do
         substituteInPlace "$f" \
           --replace "org.cups.cupsd" "cups" \
-          --replace "org.cups." ""
+          --replace "org.cups." "" \
+          --replace /var/run /run
```

Comment:
Why this change?  How is the path used?

Reply:
```/var/run``` is deprecated in favour of ```/run```.

Reply:
Ah, right, I remember.  We even have a symlink on NixOS, so it seems unlikely to break anything.